docs/modules/user-guide/attachments/jsonschemas/2.1.0/devfile.json

```diff
@@ -1647,12 +1647,12 @@
       }
     },
     "variables": {
-      "description": "Map of key-value variables used for string replacement in the devfile. Values can can be referenced via {{variable-key}} to replace the corresponding value in string fields in the devfile. Replacement cannot be used for\n\n - schemaVersion, metadata, parent source  - element identifiers, e.g. command id, component name, endpoint name, project name  - references to identifiers, e.g. in events, a command's component, container's volume mount name  - string enums, e.g. command group kind, endpoint exposure",
+      "description": "Map of key-value variables used for string replacement in the devfile. Values can be referenced via {{variable-key}} to replace the corresponding value in string fields in the devfile. Replacement cannot be used for\n\n - schemaVersion, metadata, parent source  - element identifiers, e.g. command id, component name, endpoint name, project name  - references to identifiers, e.g. in events, a command's component, container's volume mount name  - string enums, e.g. command group kind, endpoint exposure",
```

Comment:
good catch. If you dont mind, could you pls open a PR here on the devfile/api repo https://github.com/devfile/api/blob/main/pkg/apis/workspaces/v1alpha2/devworkspacetemplate_spec.go#L17 and the devfile/docs file will be updated via a workflow https://github.com/devfile/api/blob/main/.github/workflows/ci.yaml#L76

Reply:
Sure...opened here:  https://github.com/devfile/api/pull/508